### PR TITLE
Add note for custom OS repositories

### PR DESCRIPTION
**Pull request by segalaj**
- only repositories that list files are supported

### CODE REVIEW

File: admin/en/source/pages/manage-resources/populate-db-OS.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ Open source operating system versions are taken from the official repository mir
 
 To enable UForge to generate images based on the operating system it needs all the meta-data of the packages comprising the operating system. This meta-data includes the location in the storage of the package as well as dependency information that is used during generation. Furthermore, certain specific UForge packages must be populated for this operating system.
 
-.. note:: Custom repositories are supported in UForge. They are treated like other OS packages.
+.. note:: Custom repositories are supported in UForge. They are treated like other OS packages. ``Directory listings`` configuration for the repository has to be enabled on the server.
```
Comment (emuus) [inline admin/en/source/pages/manage-resources/populate-db-OS.rst]: I"m guessing the link with the last sentence is : In order for custom repositories to be included, "Directory listings" option for the repo ....

Comment (cinlloc) [reply admin/en/source/pages/manage-resources/populate-db-OS.rst]: What about using parenthesis to do the link? 

> .. note:: Custom repositories are supported in UForge (``Directory listings`` must be enabled on repository webserver). They are treated like other OS packages.

Comment (emuus) [reply admin/en/source/pages/manage-resources/populate-db-OS.rst]: > What about using parenthesis to do the link?

Parenthesis should be used for non-essential info. If an option needs to be enables to support something, then IMHO this is essential info

Comment (cinlloc) [reply admin/en/source/pages/manage-resources/populate-db-OS.rst]: > .. note:: Custom repositories are supported in UForge, only if ``Directory listings`` configuration is enabled. They are treated like other OS packages.

**Comment by emuus** [comment]
ok for me

**Pull request merged**